src/main/scala/geotrellis/sdg/PopulationNearRoadsJob.scala

```diff
@@ -123,6 +116,60 @@ class PopulationNearRoadsJob(
     ContextRDD(rdd, md)
   }
 
+  def forgottenLayerTiles(outputUri: URI): Unit = {
+    import spark.implicits._
+    val maxZoom = 12
+    val pixelsPerTile = 8
+    val wmLayoutScheme = ZoomedLayoutScheme(WebMercator, pixelsPerTile)
+    val wmLayout: LayoutDefinition = wmLayoutScheme.levelForZoom(maxZoom).layout
+    val latLngToWebMercator = Transform(LatLng, WebMercator)
+
+    val gridPointsRdd: RDD[(Long, Long, Double)] = forgottenLayer.flatMap {
+      case (key: SpatialKey, tile: Tile) => {
+        val tileExtent = key.extent(layout)
+        val re = RasterExtent(tileExtent, tile)
+        val data = for {
+          col <- Iterator.range(0, tile.cols)
+          row <- Iterator.range(0, tile.rows)
+        } yield {
+          val v = tile.getDouble(col, row)
+          if (isData(v)) {
```

Comment:
Code style alert: you can inline the if check and avoid the flatMap here:

```scala
        for {
          col <- Iterator.range(0, tile.cols)
          row <- Iterator.range(0, tile.rows)
          v = tile.getDouble(col, row)
          if isData(v)
        } yield {
          val (lon, lat) = re.gridToMap(col, row)
          val point = Point(lon, lat)
          val wmPoint = point.reproject(latLngToWebMercator)
          val (x, y) = wmLayout.mapToGrid(wmPoint)
          (x, y, v)
        }
```

Reply:
TY. I screwed up the syntax when I attempted that earlier and never went back to it once I got it working.